### PR TITLE
Archiving Old Samples

### PR DESCRIPTION
**Pull request by promisinganuj**
# Type of PR

- Documentation changes

## Purpose

- This PR is to archive old samples that are no longer maintained.
- Once this PR is merged, all the active samples would be retested and updated to use the latest tech stack.

## Does this introduce a breaking change? If yes, details on what can break

No, the changes are mostly textual so shouldn't break any active sample.

## Author pre-publish checklist

- [x] Added test to prove my fix is effective or new feature works
- [x] No PII in logs
- [x] Made corresponding changes to the documentation

## Validation steps

- ...

## Issues Closed or Referenced

- Closes #issue_number

- References #issue_number

### CODE REVIEW

**Review by ckittel — APPROVED** [review]
Thanks for archiving these items.  We'll ignore them when evaluating our Microsoft Entra branding requirements.

When you get to doing update passes, can you please prioritize `e2e_samples/parking_sensors` for both Entra updates and freshness updates, as this is the only sample currently linked to from the Azure Architecture Center.

**Comment by promisinganuj** [comment]
Hi @ckittel , I have pushed the "Microsoft Entra ID branding" changes as well in this PR. And it includes both live and archived samples. Can I please request you to take a closer look and highlight any discrepancies?

Regarding the freshness, we are currently testing the sample but given the complexity, it is going to take some time. And it would be a separate PR.

CC: @devlace

File: e2e_samples/parking_sensors/scripts/deploy_infrastructure.sh [hunk 1]
```diff
@@ -205,7 +205,7 @@ databricks_host=https://$(echo "$arm_output" | jq -r '.properties.outputs.databr
 databricks_workspace_resource_id=$(echo "$arm_output" | jq -r '.properties.outputs.databricks_id.value')
 databricks_aad_token=$(az account get-access-token --resource 2ff814a6-3304-4ab8-85cb-cd0e6f879c1d --output json | jq -r .accessToken) # Databricks app global id
```
Comment (ckittel) [inline e2e_samples/parking_sensors/scripts/deploy_infrastructure.sh:206]: In a future PR, please adjust even things like this.  `databricks_aad_token` -> `databricks_access_token`

I won't block the PR from merging over things like this, but would be good to finish getting cleaned up.

**Review by ckittel — APPROVED** [review]
THANK YOU for making the changes and archiving all that content.  `parking_sensors` was the item on our list, and this I consider resolved.  I have one suggestion below, but it's not a blocker for merging.  Maybe stuff like this can be addressed in a future pass when you are doing freshness updates.

**Review by devlace — APPROVED** [review]

**Pull request merged**